send_serial.py

```diff
@@ -1,16 +1,20 @@
 import serial  # pip install pyserial
 
-port = "COM6"  # Insert port connected to board here
+port = ""  # Insert port connected to board here
```

Comment:
This could be automatically detected or configured via command line argument.
https://pyserial.readthedocs.io/en/latest/tools.html#module-serial.tools.list_ports